### ISSUE TITLE
Selenium Chrome Webdriver gets 408 time out error

### ISSUE THREAD

**Issue by fatihtepekoy** [SeleniumHQ/selenium#8972]
## 🐛 Bug Report

Problem is getting HTTP 408 error every time when we execute the test. When we try manually there is no any problem on the URL tried lots of time, even one of them did not receive the error.

The Url we used is directed 2 times due to authorization  (LDAP login)  then comes the main page we need. I think I am facing this issue in the link below, but I could not find any solution

[https://sny.no/2018/08/webdriver-keepalive](url)

Also, there is an update in selenium change log but I dont know how to use it. 

[https://github.com/SeleniumHQ/selenium/blob/trunk/dotnet/CHANGELOG](url)
ctrl+f -> "IsKeepAliveEnabled " v3.14.0     
Thanks in advance

## Fail Scenario:
-User/pass is entered
-Click the enter botton
-Get the 408 error too quickly (20ms). (It is weird because I dont think any service configure its timeout lower than 20 milliseconds)
-Additionally backend login service does not receive any login attempt from the frontend login service. 

 ##Success scenario; (Executed on the same setup)
Success scenario; (Executed on the same setup)

refresh the same page opened by selenium
-enter User/pass
-Click the enter button
-Successfully pass the test, I mean log-in is successful, not get any error. 
-In backend login service log attempt logs can be seen 

As you see from the above test, there is not any mistake for finding username, password and enter elements, the difference is who load the page. If the selenium driver loads the url, it gets a 408 error. If refresh same page then try to login, login is successful.

I am sure now the problem related to http headers. If driver.get("loginUrl") then login error with 408 code. If I refresh the page then enter same user/pass on same the browser then it is successful.

## Expected behavior

to login successful like human clicks the enter button. 

## Test script or set of commands reproducing this issue

    String user = "myuser"
    WebElement userElement = driver.findElement(By.cssSelector(USERNAME));
    sendKeys(userElement, user);
    String pass = "myPass"
    WebElement passElement = driver.findElement(By.cssSelector(PASSWORD));
    sendKeys(passElement, pass);
    Thread.sleep(60000); // put it here a sleep, Try to enter manually before webdriver click
    driver.findElement(By.xpath(INPUT_VALUE_ENTER)).click();

## Environment

OS: Windows10
Browser:  Chrome
Browser version: 86.0.4240.183
Browser Driver version: 86.0.4240.183
Language Bindings version: Java 8
Selenium Grid version (if applicable): selenium-chrome-driver in pom version is 3.141.59

**Comment by diemol** [comment]
@fatihtepekoy, how can we reproduce this? Could you please provide a script with the latest Selenium WebDriver release for Java? (At this moment, Selenium 4 Alpha 7).

Also, have you tried with Firefox? Do you get the same results?

**Comment by fatihtepekoy** [comment]
> @fatihtepekoy, how can we reproduce this? Could you please provide a script with the latest Selenium WebDriver release for Java? (At this moment, Selenium 4 Alpha 7).
> 
> Also, have you tried with Firefox? Do you get the same results?

@diemol even me can not reproduce this scenario, it is only seen on prod system. So I dont have a script to reproduce this. 
There is just one difference between Prod login page and test login page. The difference is the prod login page works with LDAP auth., so the login http requests are handled differently.

Do you have any idea in order to find out why this happens, or a check list to follow for trouble shooting?

**Comment by diemol** [comment]
Well, if there is no way for us to reproduce it, we cannot triage this as an issue.

On the other hand, for you to debug the issue, you can:
- Try out a manual test on a browser without any plugins, cookies, logged users, perhaps in incognito mode.
- Try different browsers. Does the issue also happen in Firefox or Edge Chromium or Safari?
- Join the community Slack/IRC channels, where other folks can help you debug the issue and give more ideas. https://www.selenium.dev/support/

Having said that, I'll close this issue since there is no way to reproduce it. But if you find one, feel free to reopen or create a new issue by providing the information required on the template.

**Comment by fatihtepekoy** [comment]
@diemol 
- I tried to reproduce it on manual browser and selenium controlled browser and exactly what you said (with options --disable-cache, --no-sandbox, --incognito), both tests are successful. With same options if just selenium driver loads page, then it fails. 
- I will try it soon using different browser, then I will update here. 
- There is almost no information about this selenium 408 error, I think I found a new bug. 
- I captured tcpdump for fail and success, I will update here if I found something.
